src/main/java/org/folio/service/fund/FundService.java

```diff
@@ -9,4 +10,6 @@
 public interface FundService {
   Future<Fund> getFundById(String fundId, DBConn conn);
   Future<List<Fund>> getFundsByIds(List<String> ids, DBConn conn);
+  Future<Void> updateFund(Fund fund, RequestContext requestContext);
+  Future<Void> updateFundsWithMinChange(List<Fund> fund, DBConn conn);
```

Comment:
this method is universal, can update any field in Fund object, so can make simple updateFunds() name, also param should be named as funds

Reply:
I saw that actual implementation has some steps before updating fund https://github.com/folio-org/mod-finance-storage/blob/master/src/main/java/org/folio/rest/impl/FundAPI.java#L71, but this method not use such check before updating fund. To avoid to use this method for big changes I used specific method name. Need to think, I would agree to use updateFunds method name, what do you think in this case

Reply:
yes lets name simply to updateFunds